R/zchunk_L232.industry_USA.R

```diff
@@ -50,157 +37,280 @@ module_gcam.usa_L232.industry_USA_DISABLED <- function(command, ...) {
+      ungroup %>% filter(year %in% BASE_YEARS) %>%
+      rename(region = state) %>%
+      left_join_keep_first_only(calibrated_techs, by = c("sector", "fuel")) %>%
+      rename(stub.technology = technology) ->
+      L232.in_EJ_state_indenergy_F_Yh
+    L232.in_EJ_state_indenergy_F_Yh %>% select(one_of(LEVEL2_DATA_NAMES[["StubTechYr"]], "value")) %>%
+      left_join_keep_first_only(select(A32.globaltech_eff, subsector, technology, minicam.energy.input),
+                                by = c("subsector", "stub.technology" = "technology")) %>%
+      mutate(calibrated.value = round(value, energy.DIGITS_CALOUTPUT),
+             share.weight.year = year,
+             tech.share.weight = if_else(calibrated.value > 0, 1, 0)) %>%
+      group_by(region, supplysector, subsector, year) %>% mutate(x = sum(calibrated.value)) %>%
+      # ^^ sets up variable (x) for defining subsector shareweight
+      mutate(subs.share.weight = if_else(x > 0, 1, 0)) %>% ungroup %>%
+      select(one_of(LEVEL2_DATA_NAMES[["StubTechCalInput"]])) ->
+      L232.StubTechCalInput_indenergy_USA  ## OUTPUT
+
+    # get calibrated input of industrial feedstock technologies
+    L132.in_EJ_state_indfeed_F %>%
+      complete(nesting(state, sector, fuel), year = c(year, BASE_YEARS)) %>%
+      group_by(state, sector, fuel) %>% mutate(value = approx_fun(year, value)) %>%
+      ungroup %>% filter(year %in% BASE_YEARS) %>%
+      rename(region = state) %>%
+      left_join_keep_first_only(calibrated_techs, by = c("sector", "fuel")) %>%
+      select(-calibration, - secondary.output) %>%
+      rename(stub.technology = technology) ->
+      L232.in_EJ_state_indfeed_F_Yh
+    L232.in_EJ_state_indfeed_F_Yh %>% select(one_of(LEVEL2_DATA_NAMES[["StubTechYr"]], "value")) %>%
+      left_join_keep_first_only(select(A32.globaltech_eff, subsector, technology, minicam.energy.input),
+                                by = c("subsector", "stub.technology" = "technology")) %>%
+      mutate(calibrated.value = round(value, energy.DIGITS_CALOUTPUT),
+             share.weight.year = year,
+             tech.share.weight = if_else(calibrated.value > 0, 1, 0)) %>%
+      group_by(region, supplysector, subsector, year) %>% mutate(x = sum(calibrated.value)) %>%
+      # ^^ sets up variable (x) for defining subsector shareweight
+      mutate(subs.share.weight = if_else(x > 0, 1, 0)) %>% ungroup %>%
+      select(one_of(LEVEL2_DATA_NAMES[["StubTechCalInput"]])) ->
+      L232.StubTechCalInput_indfeed_USA  ## OUTPUT
+
+    # get industrial sector calibrated output
+    A32.globaltech_eff %>%
+      complete(nesting(supplysector, subsector, technology, minicam.energy.input, secondary.output),
+               year = c(year, BASE_YEARS)) %>%
+      group_by(supplysector, subsector, technology, minicam.energy.input, secondary.output) %>%
+      mutate(value = approx_fun(year, value)) %>% ungroup %>%  filter(year %in% BASE_YEARS) %>%
```

Comment:
Getting a bit dense; can you put `filter` and `mutate` on their own lines please?